api/api.go

```diff
@@ -414,8 +414,12 @@ type CloudBackupStatus struct {
 	OpType CloudBackupOpType
 	// State indicates if the op is currently active/done/failed
 	Status CloudBackupStatusType
-	// BytesDone indicates total Bytes uploaded/downloaded
+	// BytesDone indicates Bytes uploaded/downloaded so far
 	BytesDone uint64
+	// BytesTotal is the total number of bytes being transferred
```

Comment:
These were added to CloudBackupInfo, needs to be removed from there
https://github.com/libopenstorage/openstorage/blob/9afeef4311981abcba1c723997d09ad515a0e82f/api/api.go#L329

Reply:
thanks, will remove from there.